lib/cli.js

```diff
@@ -182,6 +183,19 @@ const cli = {
 
             log.info(JSON.stringify(fileConfig, null, "  "));
             return 0;
+        } else if (currentOptions.info) {
+            return new Promise(resolve => {
+                log.info("\n Environment Information:-");
+                envinfo
+                    .run({
+                        System: ["OS", "CPU"],
+                        Binaries: ["Node", "Yarn", "npm"],
+                        npmPackages: "eslint-{config,plugin}-*",
```

Comment:
How should the glob be updated so as to show up local `eslint` version :thinking:

Reply:
@aladdin-add ?

Reply:
npmPackages: ["eslint", "eslint-{config,plugin}-*"]

maybe it works -- I'm not very familiar with the package `envinfo`

Reply:
No, it doesn't seem to work.

Reply:
@kaicataldo @platinumazure thoughts :thinking:

Reply:
I think we should write our own code for this instead of relying on `envinfo`. See [this comment](https://github.com/eslint/eslint/pull/12060#issuecomment-519339119) for more details.